.gitignore

```diff
@@ -1,2 +1,6 @@
 out\
 .idea\
+/.idea/
+/lib/
```

Comment:
first slash is not required

Reply:
Let me leave it there as this was introduced by IDE